specs/SelectMulti.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { mount, Wrapper } from '@vue/test-utils'
+
+import SelectMulti from '../src/SelectMulti.vue'
+
+describe('SelectMulti', () => {
```

Comment:
The coverage is truly abysmal so far, but the most important thing will be having a spec runner that devs can now easily add specs to

![Screen Shot 2020-12-16 at 3 45 09 AM](https://user-images.githubusercontent.com/14101491/102325479-304a2b00-3f51-11eb-9b8a-120a0a9c8fe5.png)